gbpservice/neutron/nsf/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -0,0 +1,433 @@
+# One Convergence, Inc. CONFIDENTIAL
+# Copyright (c) 2012-2016, One Convergence, Inc., USA
+# All Rights Reserved.
+#
+# All information contained herein is, and remains the property of
+# One Convergence, Inc. and its suppliers, if any. The intellectual and
+# technical concepts contained herein are proprietary to One Convergence,
+# Inc. and its suppliers.
+#
+# Dissemination of this information or reproduction of this material is
+# strictly forbidden unless prior written permission is obtained from
+# One Convergence, Inc., USA
+
+import ast
+import json
+import requests
+
+from oslo_config import cfg
+from oslo_log import log as logging
+
+from neutron import context
+
+from gbpservice.neutron.nsf.configurator.lib import fw_constants as const
+
+LOG = logging.getLogger(__name__)
+
+
+class FwGenericConfigDriver(object):
+    """
+    Driver class for implementing firewall configuration
+    requests from Orchestrator.
+    """
+
+    def __init__(self):
+        self.timeout = cfg.CONF.rest_timeout
+
+    def configure_interfaces(self,  ev):
+        pass
+
+    def clear_interfaces(self, ev):
+        pass
+
+    def configure_license(self, ev):
+        pass
+
+    def release_license(self, ev):
+        pass
+
+    def configure_source_routes(self, ev):
```

Comment:
The event object should not be exposed to the drivers. Move the event extraction in the agent code and pass arguments to driver.